.github/workflows/main.yaml

```diff
@@ -18,16 +18,45 @@ on:
           - peak
 env:
   LOAD_SIMULATION_TYPE: ${{ github.event.inputs.TRAFFIC_LOAD || 'standard' }}
+  PATIENT_COUNT: ${{ '10' }}
```

Comment:
you dont need this env variable since you are exporting shell variable

Reply:
Removed the default PATIENT_COUNT